### PR TITLE
action: add action for nvidia driver toolkit related update

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
Update nvidia driver toolkit tag on both installer and harvester part when we tag

### CODE REVIEW

File: .github/workflows/installer-pr.yml [hunk 1]
```diff
@@ -95,3 +98,59 @@ jobs:
...
+          separator: '-'
+      - name: Download container-diff result
+        uses: actions/download-artifact@v3
+        with:
+          name: diff-result
+      - name: Read diff result
+        id: read
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: ./diff-result.txt
```
Comment (bk201) [inline .github/workflows/installer-pr.yml]: The diff result is about the harvester-os image, do we need this in harvester/harvester PR?

File: .github/workflows/installer-pr.yml [hunk 2]
```diff
@@ -95,3 +98,59 @@ jobs:
...
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: ${{ secrets.BOT_TOKEN }}
+          commit-message: Bump nvidia-driver-toolkit ${{ github.ref_name }}
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          signoff: false
+          branch: bump-os-${{ github.ref_name }}
+          delete-branch: true
+          title: 'Bump OS ${{ github.ref_name }}'
```
Comment (bk201) [inline .github/workflows/installer-pr.yml]: suggest changing the branch name and title to "Bump nvidia-driver-toolkit addon"

Comment (Vicente-Cheng) [reply .github/workflows/installer-pr.yml]: Nice catch... fixed

File: .github/workflows/installer-pr.yml (hunk 1 above)
Comment (Vicente-Cheng) [reply .github/workflows/installer-pr.yml]: Nice catch... fixed!

**Review by bk201 — APPROVED** [review]
lgtm, thanks!

**Review by ibrokethecloud — COMMENTED** [review]
lgtm. thanks.

**Pull request merged**